### PR TITLE
Bump remoting to `3071.v7e9b 0dc08466` (both parent image tag and incoming release to `3071.v7e9b 0dc08466-1`)

### PR DESCRIPTION
**Pull request by dduportal**
THis PR bumps remoting to `3071.v7e9b 0dc08466`.

It introduces the following changes:

- Bump parent image to `3071.v7e9b 0dc08466-1` (suffix `-1`) - https://github.com/jenkinsci/docker-agent/releases/tag/3071.v7e9b_0dc08466-1
- Prepare incoming release of `3071.v7e9b 0dc08466-1`
- Cherry pick chore changes from `jdk8` branch:
  - Testing framework: Bump bats version to `1.8.2`
  - Simplify Docker Bake to ease updates of parent image vs. current image version tags

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [-] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Comment by dduportal** [comment]
Once merged, a release `3071.v7e9b 0dc08466-1` can be created from https://github.com/jenkinsci/docker-inbound-agent/releases/tag/untagged-bb48f0a106cab3677787 by any maintainer to finally fix https://github.com/jenkinsci/docker-inbound-agent/issues/271

**Review by timja — APPROVED** [review]

**Pull request merged**
